ai-platform-unified/notebooks/ucaip_customjob_image.ipynb

```diff
@@ -0,0 +1,2540 @@
+{
```

Comment:
<p>The "Enable the AI Platform APIS..." link contains <code>ml.googleapis.com</code>  - I think this should be replaced with <code>aiplatform.googleapis.com</code> , as this is the service address for the AI Platform (Unified) API.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/240/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>